code/game/objects/items/mountable_frames/trophy_mount.dm

```diff
@@ -48,13 +48,12 @@
 /obj/item/mounted/frame/trophy_mount/examine(mob/user)
 	..()
 	if(held_item)
-		held_item.examine(user, show_name = FALSE)
+		held_item.examine(user, FALSE)
```

Comment:
You're only passing two arguments now, so this is broken.

Reply:
In what way?
`/obj/item/examine()` only has two parameters, and it manually places the arguments in its parent call so there wouldn't be any issues of `FALSE` being passed to the `size` parameter or such like that.

Reply:
Wait what? Oh God I didn't even notice that that's even worse.